src/lib/UniversalDiscovery/Event/Subscriber/RichTextEmbedAllowedContentTypes.php

```diff
@@ -42,16 +42,16 @@ public function __construct(
     /**
      * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
      */
-    private function getAllowedContentTypesIdentifiers(): array
+    private function getAllowedContentTypesIdentifiers(array $contentTypesAllowedViaConfig): ?array
```

Comment:
Why is it needed to return `null` value?

The code in this PR that uses this method converts `null` back into an empty array anyway :confused:

Reply:
@Steveb-p as far as I remember, method returns:
 - array (allowed identifiers),
 - empty array (no content types allowed),
 - null (all content types allowed).